exercises/practice/twelve-days/.meta/src/TwelveDays.example.elm

```diff
@@ -1,16 +1,9 @@
 module TwelveDays exposing (recite)
 
-import List exposing (map, range)
-import String exposing (concat)
-
 
 recite : Int -> Int -> List String
 recite start stop =
-    if withinSong start stop then
-        map lyric (range start stop)
-
-    else
-        Debug.todo "I would make up verses if I could...sorry about this"
+    List.map lyric (List.range start stop)
```

Comment:
There is nothing in the instructions about what to do with invalid input. Maybe we should open a PR on the problem specifications repo?

Reply:
I don't think it's necessary, there are other exercises that practice input validation, this is a simple one early in the lreaning tree.